lib/change_stream.js

```diff
@@ -19,19 +25,19 @@ const CHANGE_DOMAIN_TYPES = {
 
 /**
  * @typedef ResumeToken
+ * @property {string|Binary} _data type depends on the MongoDB versions
  * @description Represents the logical starting point for a new or resuming {@link ChangeStream} on the server.
  * @see https://docs.mongodb.com/master/changeStreams/#change-stream-resume-token
  */
 
 /**
- * @typedef OperationTime
+ * @typedef {import('bson').Timestamp} OperationTime
  * @description Represents a specific point in time on a server. Can be retrieved by using {@link Db#command}
  * @see https://docs.mongodb.com/manual/reference/method/db.runCommand/#response
  */
 
 /**
- * @typedef ChangeStreamOptions
- * @description Options that can be passed to a ChangeStream. Note that startAfter, resumeAfter, and startAtOperationTime are all mutually exclusive, and the server will error if more than one is specified.
+ * @typedef ChangeStreamOptions Options that can be passed to a ChangeStream. Note that startAfter, resumeAfter, and startAtOperationTime are all mutually exclusive, and the server will error if more than one is specified.
```

Comment:
JSDoc plugin for eslint demands typedef be followed immediately by `@property` (if any exist) I realize now I can move description to the end of the property list and it will be happy. However, I see that VSCode only picks up the description for hover docs if the description follows the typedef like this, that certainly doesn't have to be the rule we live by though